package.json

```diff
@@ -74,29 +74,29 @@
   "devDependencies": {
     "@commitlint/cli": "^17.3.0",
     "@commitlint/config-conventional": "^17.3.0",
-    "@loopback/boot": "^5.0.9",
-    "@loopback/build": "^9.0.9",
-    "@loopback/context": "^5.0.9",
-    "@loopback/repository": "^5.1.4",
-    "@loopback/testlab": "^5.0.9",
+    "@loopback/boot": "^5.0.10",
+    "@loopback/build": "^9.0.10",
+    "@loopback/context": "^5.0.10",
+    "@loopback/repository": "^5.1.5",
+    "@loopback/testlab": "^5.0.10",
     "@loopback/tslint-config": "^2.1.0",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/commit-analyzer": "^9.0.2",
     "@semantic-release/git": "^10.0.1",
     "@semantic-release/npm": "^9.0.1",
     "@semantic-release/release-notes-generator": "^10.0.3",
     "@types/lodash": "^4.14.191",
-    "@types/node": "^14.18.36",
-    "@typescript-eslint/eslint-plugin": "^5.54.0",
-    "@typescript-eslint/parser": "^5.54.0",
+    "@types/node": "^14.18.42",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
+    "@typescript-eslint/parser": "^5.58.0",
     "commitizen": "^4.2.4",
     "cz-conventional-changelog": "^3.3.0",
     "cz-customizable": "^6.3.0",
-    "eslint": "^8.35.0",
-    "eslint-config-prettier": "^8.7.0",
+    "eslint": "^8.38.0",
+    "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-eslint-plugin": "^5.0.8",
     "eslint-plugin-mocha": "^10.1.0",
-    "fs-extra": "^10.1.0",
+    "fs-extra": "^11.1.1",
```

Comment:
Confirm if this major version update isn't breaking anything we're using.

Reply:
This package is not being used in soft-delete so can be removed.

Was added in this commit: https://github.com/sourcefuse/loopback4-soft-delete/commit/6910f78a207330a820d5535dfdacdfb17ff9587e